### ISSUE TITLE
Issue with uploading to iOS device

### ISSUE THREAD

**Issue by ScottLee97** [Meherdeep/agora-group-calling#5]
Hi Meher, 

When I am trying to run this app on my iOS device using Xcode, I received the following error:

```
 Module 'agora_rtc_engine' not found
```
Do you know how to go about this?

And can I check with you which flutter and dart version you are using?

Thanks!

**Comment by Meherdeep** [comment]
@ScottLee97 Can you try removing the podfile from the iOS folder and then rerun it? 

You can remove it by writing this line in the terminal: `rm -rf podfile`

**Comment by ScottLee97** [comment]
Hi Meher, thanks for the prompt reply. I tried the line and also the following code:

rm ios/Podfile
rm ios/Podfile.lock
rm -rf ios/Pods
rm -rf ios/Runner.xcworkspace
flutter clean
flutter build ios

But the problem still persisted. 

I realized later that the problem is because I was running the app using runner.xcproj instead of runner.xcworkspace. Once I switch to running on runner.xcworkspace, the problem was solved. Thank you for your help!
